src/Welcome.mdx

```diff
@@ -1,16 +1,3 @@
-import { Meta } from "@storybook/addon-docs";
-
-<Meta
-  title="Welcome"
-  parameters = {{
-     previewTabs: {
-      'storybook/docs/panel': { index: -1 },
-      'canvas': { disable: true }
-     },
-    viewMode: 'docs'
-  }}
-/>
-
 # conda-store-ui component documentation
```

Comment:
This is just a plain markdown file now. We're not parsing as mdx. Can we just use `.md` for the file extension? Or is there some reason to keep it as `.mdx`?

Reply:
Apparently you have to use `.mdx`. I changed this to `.md` and updated the Storybook config. It didn't work.